src/js/components/FileInput/FileInput.js

```diff
@@ -328,6 +330,7 @@ const FileInput = forwardRef(
                   icon={<RemoveIcon />}
                   hoverIndicator
                   onClick={(event) => {
+                    if (onClickRemove) if (!onClickRemove()) return;
```

Comment:
It should be the same as `if (onClickRemove && !onClickRemove()) return;`? :)

Reply:
I think the function will want to know more about which file is being removed.